app/client/components/HelloWorld.js

```diff
@@ -0,0 +1,38 @@
+
+import React from 'react';
+
+export default class HelloWorld extends React.Component {
+
+    static propTypes = {
+        text: React.PropTypes.string,
+        name: React.PropTypes.string,
+    }
+
+    static defaultProps = {
+        text: 'i am text',
+        name: 'Kristina',
+    }
+
+    render() {
+        var { text, name } = this.props;
+
+        var background = {
+            backgroundColor: 'lime',
+        };
+
+        var nameStyle = {
+            color: 'tomato',
+            border: '2px solid blue'
```

Comment:
you are missing a comma `,` at the end of this line :-)
It's not an error but still an important code quality point.

You can run an automatic checking for those kind of stuff by:

`npm run lint`

on your console